src/args.rs

```diff
@@ -99,6 +99,15 @@ pub struct Args {
     )]
     pub exclude: Regex,
 
+    /// HTTP basic authentication credentials
+    #[structopt(
+        short,
+        long,
+        use_delimiter = true,
+        help = "HTTP basic authentication credentials, comma separated as \"username,password,host\""
+    )]
+    pub auth: Vec<String>,
```

Comment:
I know this is what was agreed upon, but doesn't it make more sense to have three separate options?

I'd rather type `suckit --user UN --password PW --host H` than `suckit --auth UN,PW,H` because the latter one implies
looking at the man, and also adding some parsing, be it from you or structopt. For example, what happens when someone's password contains a comma? What if they don't escape it? Right now, the authentication parsing would work, and the request would fail, making it a strange behavior considering the user DID enter their password correctly

Reply:
I think you're right that it would be more simple to split those out after pulling it together with delimiters, even if we do add a separate option that loads them in comma-separated pairs from a file. I'll split this into separate options, thanks!

Reply:
> I'd rather type `suckit --user UN --password PW --host H` than `suckit --auth UN,PW,H`

So if I have:
abc.xy:
- username: blabla
- password: bloublou

and
def.zz:
- username: cafe
- password: chocolatine

You want something like: ``suckit --user blabla,caf --password bloublou,chocolatine --host abc.xy,def.zz``. Imo this is kind of weird

Reply:
Sure, I agree that this is kind of weird. I like the idea where commas are not used. Also, if people want to use multiple logins they should use a file, so it might not be an issue to make it "hard" or impractical to use many logins from the command line

Reply:
Thanks for the discussion here! Updated it to use space separation and chunking so that users can pass `suckit --auth u1 p1 h1 u2 '' h2` for two sets of credentials with optionally ignored or quoted values